src/seedu/addressbook/parser/Parser.java

```diff
@@ -233,6 +296,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * Parses arguments in the context of the find person command.
      *
      * @param args full command args string
+     *
```

Comment:
There are a lot of unnecessary blank lines. Please check coding standard for javadoc comments

Reply:
Noted. Thank you! It is my bad habit of separating components of the code by blank lines. I will update in the level3/4.